model/src/main/proto/profile.proto

```diff
@@ -136,3 +136,5 @@ enum AudioLanguage {
   CHINESE_AUDIO_LANGUAGE =  5;
   BRAZILIAN_PORTUGUESE_LANGUAGE = 6;
 }
+
```

Comment:
Nit: just one EOF newline.

Reply:
Updated.

Reply:
Per the latest changes, it seems like there are still 3 newlines here.